tests/receivers/adopted/redisreceiver/testdata/client/init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Beginning redis client"
+
+# Checking for the connection, return "PONG" if succeded
+redis-cli -h redis-server ping
+
+# Setting key value pair in the redis server from the client, return "OK" if succeded
+redis-cli -h redis-server set tempkey tempvalue
+
+echo "redis client started"
```

Comment:
Ideally this would have a dedicated loop to populate and query redis (set/get) as a client.